nilearn/glm/first_level/design_matrix.py

```diff
@@ -301,7 +302,8 @@ def make_first_level_design_matrix(
         In case of FIR design, yields the array of delays used in the FIR
         model (in scans). Default=[0].
 
-    add_regs : array of shape(n_frames, n_add_reg) or pandas DataFrame, optional
+    add_regs : array of shape(n_frames, n_add_reg) or
+               pandas DataFrame, optional
```

Comment:
Splitting the data type across lines like this doesn't seem to render as expected:

![image](https://user-images.githubusercontent.com/8228902/129585750-152c0036-03f9-4f4d-8386-4e15257abb99.png)